pandas/core/window/rolling.py

```diff
@@ -922,11 +922,11 @@ class Window(_Window):
     * ``blackmanharris``
     * ``nuttall``
     * ``barthann``
-    * ``kaiser`` (needs beta)
-    * ``gaussian`` (needs std)
-    * ``general_gaussian`` (needs power, width)
-    * ``slepian`` (needs width)
-    * ``exponential`` (needs tau), center is set to None.
+    * ``kaiser`` (needs beta in operation argument)
+    * ``gaussian`` (needs std in operation argument)
+    * ``general_gaussian`` (needs power, width in operation argument)
+    * ``slepian`` (needs width in operation argument)
+    * ``exponential`` (needs tau in operation argument), center is set to None.
```

Comment:
I'm wondering if it may be helpful to also add some additional explanation below (something like: "Certain window types require that a scale parameter be passed...") since this is a fairly confusing situation, and then call attention to the Gaussian case in the examples section. Thoughts @wchen928 @MarcoGorelli?

Reply:
How about we change to the following and add one more sentence before we list all the examples. 

```suggestion
    * ``exponential`` (need parameter: tau), center is set to None.
```
Additional sentence: Certain window types require a scale parameter to be passed. Please see the third example below to see how to add the additional parameters.

Reply:
> Please see the third example below on how to add the additional parameters.

Sure, sounds good to me

Reply:
Great. I will wait for @dsaxton 's opinion and before editing it.

Reply:
Works for me too 👍